### PR TITLE
Add necessary files for the runtime.management module

### PR DESCRIPTION
**Pull request by gabilang**
## Automation tests
 - Unit tests 
   > Code coverage information
 - Integration tests
   > Details about the test cases and coverage

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes/no
 - Ran FindSecurityBugs plugin and verified report? yes/no
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes/no

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+}
+
+release {
+    failOnPublishNeeded = false
+
+    buildTasks = ['build']
+    versionPropertyFile = 'gradle.properties'
+    tagTemplate = 'v$version'
+
+//    git {
```
Comment (warunalakshitha) [inline build.gradle]: shall we remove commented lines

File: README.md [hunk 2]
```diff
@@ -1,2 +1,68 @@
...
+  [![Github issues](https://img.shields.io/github/issues/ballerina-platform/module-ballerinai-runtime.management.svg?label=Open%20Issues)](https://github.com/ballerina-platform/module-ballerinai-runtime.management/issues)
+  [![License](https://img.shields.io/badge/License-Apache%202.0-blue.svg)](https://opensource.org/licenses/Apache-2.0)
+  [![codecov](https://codecov.io/gh/ballerina-platform/module-ballerinai-runtime.management/branch/master/graph/badge.svg)](https://codecov.io/gh/ballerina-platform/module-ballerinai-runtime.management)
+  [![GraalVM Check](https://github.com/ballerina-platform/module-ballerinai-runtime.management/actions/workflows/build-with-bal-test-graalvm.yml/badge.svg)](https://github.com/ballerina-platform/module-ballerinai-runtime.management/actions/workflows/build-with-bal-test-graalvm.yml)
+
+The Ballerina runtime.management internal module defines a service to expose runtime artifacts like deployed services, 
+configurable variables, and resources to control runtime artifacts. Once the user defines the configurations 
+inside Ballerina.toml, the management module will be imported to the user's project.
```
Comment (HindujaB) [inline README.md]:
``` suggestion
The Ballerina runtime.management internal module defines a service to expose runtime artifacts like deployed services, 
configurable variables, and resources to control the runtime artifacts. Once the user defines the configurations 
inside the `Ballerina.toml`, the runtime.management module will be imported to the user's package.
```

File: README.md [hunk 3]
```diff
@@ -1,2 +1,68 @@
...
+
+
+During initialization, the ballerina node will be registered in the integration control plane using configurations 
+in Ballerina.toml. Then the integration control plan will connect with a given node via a service the module exposes 
+to discover and control artifacts.
```
Comment (HindujaB) [inline README.md]:
``` suggestion
During initialization, the ballerina node will be registered in the integration control plane using the configurations provided 
in `Ballerina.toml`. Then the integration control plane will connect with a given ballerina node via a service that the module exposes 
to discover and control the artifacts.
```

File: ballerina/management.bal [hunk 4]
```diff
@@ -0,0 +1,101 @@
...
+
+type Module record {|
+   string name;
+   string org;
+   string version;
+|};
+
+configurable DashBoard dashboard = ?;
+
+service /management on new http:Listener(9164) {
```
Comment (HindujaB) [inline ballerina/management.bal:71]: the listener port `9164` is hard coded here. Is this based on the integration control plane configuration? Shall we move it to a constant and use it here?

File: native/src/main/java/module-info.java [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+module org.ballerinalang.stdlib.runtime.management {
+    requires io.ballerina.runtime;
+}
```
Comment (HindujaB) [inline native/src/main/java/module-info.java]:
``` suggestion
}

```

File: .github/workflows/pull-request.yml [hunk 6]
```diff
@@ -0,0 +1,54 @@
+name: PR build
+
+on:
+  pull_request:
+  push:
+    branches-ignore:
+      - main
```
Comment (HindujaB) [inline .github/workflows/pull-request.yml]:
``` suggestion
```

File: .github/workflows/pull-request.yml [hunk 7]
```diff
@@ -0,0 +1,54 @@
+name: PR build
+
+on:
+  pull_request:
```
Comment (HindujaB) [inline .github/workflows/pull-request.yml]:
``` suggestion
  pull_request:
    branches:
      - main
```

File: ballerina/management.bal (hunk 4 above)
Comment (gabilang) [reply ballerina/management.bal:71]: It'll get changed with @warunalakshitha's commits. Just added the proposed Ballerina file in the design doc.

Comment (warunalakshitha) [reply ballerina/management.bal:71]: I will change with my code

**Review by warunalakshitha — APPROVED** [review]

**Review by HindujaB — APPROVED** [review]
LGTM

**Pull request merged**